internal/scorecard/tests/olm.go

```diff
@@ -219,8 +219,45 @@ func checkCSVDescriptors(bundle *apimanifests.Bundle, r scapiv1alpha3.TestResult
 	}
 	r.Log += fmt.Sprintf("Loaded %d Custom Resources from alm-examples\n", len(crs))
 
-	for _, cr := range crs {
-		r = checkOwnedCSVDescriptors(cr, bundle.CSV, descriptor, r)
+	if descriptor == statusDescriptor {
+		for _, cr := range crs {
+			r = checkOwnedCSVStatusDescriptor(cr, bundle.CSV, r)
+		}
+	} else {
+		for _, cr := range crs {
+			r = checkOwnedCSVSpecDescriptors(cr, bundle.CSV, r)
+		}
+	}
```

Comment:
```suggestion
	for _, cr := range crs {
	        // if the descriptor is status then, check the status otherwise check the spec
	 	if descriptor == statusDescriptor {
		 	r = checkOwnedCSVStatusDescriptor(cr, bundle.CSV, r)
		} else {
		        r = checkOwnedCSVSpecDescriptors(cr, bundle.CSV, r)
		}
	}
```